Firestore/core/test/firebase/firestore/remote/grpc_stream_test.cc

```diff
@@ -102,157 +147,409 @@ class GrpcStreamTest : public testing::Test {
     return {states};
   }
 
-  bool ObserverHas(const std::string& state) const {
-    return std::find(observed_states().begin(), observed_states().end(),
-                     state) != observed_states().end();
-  }
-
-  void StartStream() {
-    worker_queue.EnqueueBlocking([&] { stream().Start(); });
-  }
-
   AsyncQueue worker_queue;
 
- private:
-  std::unique_ptr<ConnectivityMonitor> connectivity_monitor_;
-  GrpcStreamTester tester_;
+  std::unique_ptr<ConnectivityMonitor> connectivity_monitor;
+  GrpcStreamTester tester;
 
-  std::unique_ptr<Observer> observer_;
-  std::unique_ptr<GrpcStream> stream_;
+  std::unique_ptr<Observer> observer;
```

Comment:
Alternatively to expose those as public, you can keep the name observer_ and put them under protected:. Optional, which means I am not against one or the other way.